docs/built-in-functions.rst

```diff
@@ -545,6 +545,136 @@ Math
         >>> ExampleContract.foo(11, 2, 5)
         2
 
+.. py:function:: unsafe_add(x: integer, y: integer) -> integer
+
+    Add ``x`` and ``y``, without checking for overflow. ``x`` and ``y`` must both be integers of the same type. If the result exceeds the bounds of the input type, it will be wrapped.
+
+    .. code-block:: python
+
+        @external
+        @view
+        def foo(x: uint8, y: uint8) -> uint256:
```

Comment:
What's the difference between this and bar below?

Reply:
foo accepts uint8, bar accepts int8 (signed)